<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Abhi-PC" name="Test1" tests="2" failures="1" timestamp="9 Dec 2017 12:36:54 GMT" time="30.651" errors="0">
  <testcase name="basePageNavigation" time="0.074" classname="tests.TestDemo">
    <failure type="java.lang.AssertionError" message="expected [Intentional Fail Scenario] but found [Go ahead and give it full test to checkout the amazing features behind our application]">
      <![CDATA[java.lang.AssertionError: expected [Intentional Fail Scenario] but found [Go ahead and give it full test to checkout the amazing features behind our application]
	at tests.TestDemo.basePageNavigation(TestDemo.java:40)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
  <testcase name="demoPassTest" time="0.062" classname="tests.TestDemo"/>
</testsuite> <!-- Test1 -->
